SEMANTIC-VERSIONING.md

```diff
@@ -0,0 +1,137 @@
+1. **Additional merging** within `kind` **values** at zooms 13, 14, or 15 by removing `common`, `common-optional`, and `optional` **properties** or other method
+1. **Reassign** 50% or more of existing `kind` **value** into a new `kind` **value**, when kind has 10,000 or more features
+1. **Change** of <= -2 (earlier) to `min_zoom` or `max_zoom` **values** to determine when `kind` is included
+1. **Change** of >= +1 (later) to default `min_zoom` or `max_zoom` **values** to determine when `kind` is included
+1. **Adjustments** to the overall map balance (proportion of features in one layer or another, proportion of `kind`s in a single layer)
+
+#### PATCH version increments
+
+1. **Add** `optional` layer
+1. **Change** `optional` layer **name**
+1. **Remove** `optional` layer
+1. **Additional merging** within `kind` **values** at zooms 12 or less by removing `common`, `common-optional`, and `optional` **properties** or other method
+1. **Reassign** less than 50% of existing `kind` **value** into a new `kind` value, when kind has 10,000 or more features
+1. **Reassign** more than 50% of existing `kind` **value** into a new `kind` value, when kind has less than 10,000 features
+1. **Change** of -1 to default `min_zoom` or `max_zoom` **values** to determine when `kind` is included
+1. **Add** unpublicised `kind` **value**
+1. **Remove** unpublicised `kind` **value**
+1. **Correct** a regression in the API (to the last good version)
+1. **Correct** a newly added feature **property name**
+1. **Correct** a newly added `kind` **value**
+
+## Versioning Data
+
+We do not version data features, but we do attempt to indicate the data source and the feature ID as assigned by that source so customers can investigate upstream changes.
+
+### INDIVIDUAL FEATURES are not versioned
+
+#### Examples:
+
+1. **Add** new feature
+1. **Removal** of existing feature
+1. **Change** feature **name**
+1. **Change** feature **geometry**
+1. **Change** feature **property** values (including property removal if removed from original source)
+1. **Change** feature `kind` **value** (when upstream data source reclassifies them).
+1. **Change** feature `min_zoom` &/or `max_zoom` **values** (when area or other signal changes upstream)
+
+**NOTE:** It is possible to query the version of indivdual features by looking at a feature's `source` and `id` properties and performing a lookup via the source service, but that is beyond the scope of Tilezen.
+
+
+### POLITICAL GEOGRAPHY is not versioned
+
+#### Examples:
+
+1. **Major** additions, deletions to country names, borders, disputed territories, and capitals are possible and may be advertized.
+    - Generally the **Natural Earth** source we currently use for low zooms (0-8) doesn’t update frequently.
+    - **OpenStreetMap** (used for zooms 9+) updates minutely
+    - **Who’s On First** (used for zooms 12+) updates minutely.
```

Comment:
This isn't currently true, and it may be the sort of thing that's better left unspecified. Can we say that it is "constantly updating" or that it's on an "ongoing" or "regular" basis? As it stands this would give me the impression that when a feature is updated in osm, I would expect to see it minutes later.